cf-execd/cf-execd.c

```diff
@@ -623,6 +623,9 @@ static bool ScheduleRun(EvalContext *ctx, Policy **policy, GenericAgentConfig *c
 
         DetectEnvironment(ctx);
 
+        Log(LOG_LEVEL_INFO, "Re-evaluating augments" );
```

Comment:
VERBOSE please, no need to spam syslog with this.

Reply:
You said it was OK as is. So merging.